src/operon-runtime/runtime.ts

```diff
@@ -48,13 +50,13 @@ export class OperonRuntime {
    * Load an application's Operon functions, assumed to be in src/operations.ts (which is compiled to dist/operations.js).
    */
   private loadFunctions(): Promise<ModuleExports> | null {
-    const workingDirectory = process.cwd();
-    const operations = workingDirectory + "/dist/operations.js";
+    const entrypoint = this.runtimeConfig.entrypoint ?? "dist/operations.js";
```

Comment:
Didn't see that earlier, but we should set the default value in `config.ts`, like we do for the port